core/trino-server-core/src/main/provisio/trino.xml

```diff
@@ -0,0 +1,96 @@
+<runtime>
+    <!-- Target -->
+    <archive name="${project.artifactId}-${project.version}.tar.gz" hardLinkIncludes="**/*.jar" />
+
+    <!-- Notices -->
+    <fileSet to="/">
+        <directory path="${basedir}">
+            <include>NOTICE</include>
+            <include>README.txt</include>
+        </directory>
+    </fileSet>
+
+    <!-- Launcher -->
+    <artifactSet to="bin">
+        <artifact id="io.airlift:launcher:tar.gz:bin:${dep.launcher.version}">
+            <unpack />
+        </artifact>
+        <artifact id="io.airlift:launcher:tar.gz:properties:${dep.launcher.version}">
+            <unpack filter="true" />
+        </artifact>
+    </artifactSet>
+
+    <!-- Server -->
+    <artifactSet to="lib">
+        <artifact id="${project.groupId}:trino-server-main:${project.version}" />
+    </artifactSet>
+
+    <!-- Configuration Plugins -->
+    <artifactSet to="secrets-plugin/keystore-secrets-plugin">
+        <artifact id="io.airlift:secrets-keystore-plugin:zip:${dep.airlift.version}">
+            <unpack />
+        </artifact>
+    </artifactSet>
+
+    <!-- Plugins -->
+    <artifactSet to="plugin/exchange-filesystem">
+        <artifact id="${project.groupId}:trino-exchange-filesystem:zip:${project.version}">
+            <unpack />
+        </artifact>
+    </artifactSet>
+
+    <artifactSet to="plugin/exchange-hdfs">
+        <artifact id="${project.groupId}:trino-exchange-hdfs:zip:${project.version}">
+            <unpack />
+        </artifact>
+    </artifactSet>
+
+    <artifactSet to="plugin/functions-python">
+        <artifact id="${project.groupId}:trino-functions-python:zip:${project.version}">
+            <unpack />
+        </artifact>
+    </artifactSet>
+
+    <artifactSet to="plugin/geospatial">
+        <artifact id="${project.groupId}:trino-geospatial:zip:${project.version}">
+            <unpack />
+        </artifact>
+    </artifactSet>
+
+    <artifactSet to="plugin/memory">
```

Comment:
Was there a discussion already which connectors to include here? I'm wondering if tpch or faker would be useful. Faker is designed for copying other catalogs, so maybe it's an overkill.

The reasoning around this should be captured in the README - if this package is only supposed to be used for building other packages, or are there any other use cases, like testing.

Reply:
Not yet .. this PR is aimed to kick off this discussion ..